internal/cmd/operator-sdk/generate/internal/relatedimages.go

```diff
@@ -142,10 +156,8 @@ func (c *relatedImageCollector) collectedRelatedImages() []operatorsv1alpha1.Rel
 				continue
```

Comment:
If a related image with the same name/image was discovered, do nothing. It's already there.